tasks/confirm-build.js

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { execSync } from 'child_process';
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
```

Comment:
Related: is there some reason why we're using function expressions everywhere in these files, as opposed to function declarations?

I ask because it might be nice to put these copy-pasted functions at the bottom of each file, leaving the unique implementation of each task at the top, and the "do-not-edit-here" part at the bottom...but this will only work if we use function declarations, since otherwise the function implementation needs to precede invocation.

Reply:
Yes!
Makes sense, I will create a single source of truth for ignored packages and workspace packages so that all scripts can import and leverage that!